python/ray/serve/_private/application_state.py

```diff
@@ -75,11 +123,18 @@ def update(self):
 
         if self.status == ApplicationStatus.DELETING:
             mark_delete = True
+            # Application won't be deleted until all deployments get cleaned up
             for name in self.get_all_deployments():
                 if self.deployment_state_manager.get_deployment(name):
+                    logger.warn(
```

Comment:
Why is this `logger.warn` instead of `logger.info`?

Reply:
sorry, I confused myself, change it to `debug`, this func is following controller update cycle.